core/Mail.php

```diff
@@ -33,6 +33,7 @@ class Mail
     protected $bccs = [];
     protected $attachments = [];
     protected $smtpDebug = false;
+    protected $bcc;
```

Comment:
That's actually incorrect. The usage of `bcc` should be `bccs` instead.

Reply:
Thank you @sgiehl . That's a good catch. I fixed it and wrote a few unit tests to confirm it's working as expected.